adcm_client/objects.py

```diff
@@ -722,7 +722,8 @@ def run(self, **args) -> "Task":
                             args['config'][key][subkey] = config_diff[key][subkey]
                         elif not subkey and key in config_diff:
                             args['config'][key] = config_diff[key]
-
+            if 'verbose' not in args:
+                args['verbose'] = False
```

Comment:
There are several verbose options for ansible (-v, -vv, -vvv, -vvvv) What key is this option for?

I guess we need to document it somehow. Maybe in ADCM itself.

Reply:
It is documented in task only for now.